assets/src/edit-story/components/form/numeric.js

```diff
@@ -152,53 +242,27 @@ function Numeric({
       {/* type="text" is default but added here due to an a11y-related bug. See https://github.com/A11yance/aria-query/pull/42 */}
       <StyledInput
         type="text"
-        ref={ref}
+        ref={inputRef}
         placeholder={placeholder}
         prefix={prefix}
         suffix={suffix}
         label={label}
-        value={
-          isMultiple
-            ? ''
-            : `${inputValue}${dot ? DECIMAL_POINT : ''}${focused ? '' : symbol}`
-        }
+        value={isMultiple ? '' : `${inputValue}${focused ? '' : symbol}`}
         disabled={disabled}
         {...rest}
-        onChange={(evt) => {
-          const newValue = evt.target.value;
-          if (newValue === '') {
-            // Allow input to be empty
-            if (canBeEmpty) {
-              // Send empty up-stream
-              onChange('', evt);
-            } else {
-              // Just update empty value internally but keep old value upstream
-              setInputValue('');
-            }
-          } else if (newValue === '-' && canBeNegative) {
-            // Allow a single "-" if negative values are allowed
-            setInputValue(newValue);
-          } else {
-            setDot(float && newValue[newValue.length - 1] === DECIMAL_POINT);
-            const valueAsNumber = float
-              ? parseFloat(newValue)
-              : parseInt(newValue);
-            if (!isNaN(valueAsNumber)) {
-              onChange(valueAsNumber, evt);
-            }
-          }
-        }}
-        onBlur={(evt) => {
-          if (evt.target.form) {
-            evt.target.form.dispatchEvent(
-              new window.Event('submit', { cancelable: true })
-            );
+        onKeyDown={handleKeyDown}
```

Comment:
Why is this not using `useKeyDownEffect`?

Reply:
I couldn't get the `useKeyDownEffect` to trigger in the jest test, but I'll look again, I probably overlooked something.